src/huggingface_hub/keras_mixin.py

```diff
@@ -27,45 +27,27 @@
     import tensorflow as tf
 
 
-def _extract_hyperparameters_from_keras(model):
+def _create_hyperparameters_table(model):
     if model.optimizer is not None:
-        hyperparameters = dict()
-        hyperparameters["optimizer"] = model.optimizer.get_config()
-        hyperparameters[
+        optimizer_params = model.optimizer.get_config()
+        optimizer_params[
             "training_precision"
         ] = tf.keras.mixed_precision.global_policy().name
+        table = "|"
+        for key in optimizer_params.keys():
+            table += f" {key} |"
+
+        table += "\n|"
+        for key in optimizer_params.keys():
+            table += "-" * len(key)
+            table += "|"
+
+        table += "\n|"
+        for key in optimizer_params.keys():
+            table += f"{optimizer_params[key]}|"
     else:
-        hyperparameters = None
-    return hyperparameters
-
-
-def _parse_model_history(model, save_directory):
-    lines = None
-    if model.history is not None:
-        if model.history.history != {}:
-            path = os.path.join(save_directory, "history.json")
-            with open(path, "w", encoding="utf-8") as f:
-                json.dump(model.history.history, f, indent=2, sort_keys=True)
-            lines = []
-            logs = model.history.history
```

Comment:
Wait so here the training metrics section is gone too? No TensorBoard logs either?

Reply:
TensorBoard logs don't leverage this anyway.
see https://github.com/huggingface/huggingface_hub/blob/65d3c88a24b648dd65ad76d78f11d76c45a7083c/src/huggingface_hub/keras_mixin.py#L422